ruled_labels/[DRAFT]specs_monorepo.yaml

```diff
@@ -0,0 +1,187 @@
+
+name: Monorepo labels
+description: Import on 2022-08-10
+
+codes:
+  - code: A
+    name: Action 
+    description: used with GHA and trigger a certain process
+  - code: B
+    name: Dependencies 
+    description: Upgrade dependencies
+  - code: C
+    name: Contribution 
+    description: Upgrade dependencies
+  - code: D
+    name: Auditing 
+    description: Required when there were changes in the paths [CI/CD INPUT NEW FILE PATHS]. Also used to indicate the need for a documentation audit of DevRel.
+  - code: F
+    name: Fails
+    description: Change breaks some part of the code
+  - code: I
+    name: Issues 
+    description: Categorisation of issues
+  - code: T
+    name: Topics
+    description: PR/Issue is related to this topic
+
+labels:
+  - name: A0-please_review
+    description: Pull request needs code review.
+    color: "666666"
+  - name: A1-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps team is notified via matrix.
+    color: c7def8
+  - name: A2-insubstantial
+    description: Pull request requires no code review (e.g., a sub-repository hash update).
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-stale
+    description: Pull request did not receive any updates in a long time. No review needed at this stage. Close it.
+    color: F2C5F8
+  - name: A5-backport
+    description: Pull request is already reviewed well in another branch.
+    color: F8C5E7
+  - name: A6-unconfirmed
+    description: Issue might be valid, but it's not yet known.
```

Comment:
I think we should delete in this section `A0` and `A3` and instead use the fields in the projects. Anyone disagrees?
Also added `A6` here as I think it fits good to this section, any objections @alvicsam?

Reply:
No objections